website/src/docs/dragdrop.md

```diff
@@ -88,6 +88,18 @@ Optionally, specify a string of text that explains something about the upload fo
 
 Localize text that is shown to the user.
 
+### `onDragOver: undefined`
```

Comment:
`undefined` bit will end up in the ancor/hash url, so I try to avoid that. One was to deal with this is to put arguments in a list below the title, like here: https://uppy.io/docs/uppy/#uppy-log